master/usage/migrating-pools.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Migrating to a new IP Pool
+---
+
+## About migrating active pools
```

Comment:
A general bit of feedback - I think users are going to be confused about terms like "active pools" and "live pools" since they aren't really defined anywhere. I think we should standardize on a term, and ideally it should be one that we don't need to define at the top of the page.

e.g. How about instead of "About migrating active pools" it could just be "About changing IP pools" or "Changing IP pools"

instead of "...guidance on how to migrate off of a live Calico pool" we could say "instructions for how to change from one IP pool to another on a running cluster".

Reply:
Also, I think we should be careful to refer to them as "IP pools" instead of just "pools" since that is what the resource is called.